website/content/gateway/api_reference/resources/debits/debits.md

```diff
@@ -14,5 +14,5 @@ Debits support the same parameters and payment methods as [Authorizations](#auth
 Information about sender and recipient is required for debits under certain circumstances to be compliant with card scheme rules. See [Sender information](#sender_information) and [Recipient information](#recipient_information).
 
 {{% notice %}}
-**Notice:** Debits are only supported for Visa cards and funding transactions. In addition, only merchants with selected Merchant Category Codes (MCCs) and assigned Business Application Identifiers (BAIs) are able to process a debit; namely exactly those that will result in the debit being a Visa Account Funding Transaction (AFT).
+**Notice:** Only merchants with selected Merchant Category Codes (MCCs) and assigned Business Application Identifiers (BAIs) (VISA) or Transaction Type Indicators (TTIs) (Mastercard) are able to process a debit; namely exactly those that will result in the debit being a Visa or Mastercard Account Funding Transaction (AFT).
```

Comment:
```suggestion
**Notice:** Only merchants with selected Merchant Category Codes (MCCs) and assigned Business Application Identifiers (BAIs) (VISA) or Transaction Type Indicators (TTIs) (Mastercard) are able to process a debit; namely exactly those that will result in the debit being a Visa Account Funding Transaction (AFT) or Mastercard Funding Transaction (FT).
```
As we are trying to use the scheme terms, I will suggest we use Account Funding Transaction for Visa and Funding Transaction for Mastercard debits.